src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -57,8 +57,6 @@ class LedgerImpl(vm: VM, blockchainConfig: BlockchainConfig) extends Ledger with
     val blockchain = BlockchainImpl(storages)
 
     val blockExecResult = for {
-      //todo why do we require in executeBlockTransactions separate blockchain and blockchain storage
-      //todo blockchain storage is part of blockchain
```

Comment:
:thinking: 
That would be practical, however I'm not sure whether we should do because so far `Blockchain` is more of an abstract trait without implementation details such as the storages used.
Maybe we should probably add a function like `getWorldState` to the `Blockchain`, to solve this.